.gitignore

```diff
@@ -1,9 +1,152 @@
-### Ignore for NodeJs
-node_modules
```

Comment:
deleting this from gitignore will make travis also thats code we dont really need too be up here

Reply:
I added up in line 42